scripts/Tools/Makefile

```diff
@@ -873,7 +878,16 @@ endif
 
 # libcsm_share.a is in ULIBDEP, but -lcsm_share is in ULIBS rather than CLIBS,
 # so this needs to be added after creating CLIBS above
-CSMSHARELIB = $(INSTALL_SHAREDPATH)/$(COMP_INTERFACE)/$(ESMFDIR)/$(NINST_VALUE)/lib/libcsm_share.a
+ifeq ($(CIME_MODEL),cesm)
+  ifneq ($(COMP_INTERFACE), nuopc)
+    CSMSHARELIB_CPL7 = $(INSTALL_SHAREDPATH)/$(COMP_INTERFACE)/$(ESMFDIR)/$(NINST_VALUE)/lib/libcsm_share_cpl7.a
+  endif
+endif
+ifeq ($(MODEL),csm_share_cpl7)
+  CSMSHARELIB = $(CSMSHARELIB_CPL7)
```

Comment:
Can you please add a comment explaining the purpose of this unintuitive line?

Reply:
This isn't needed anymore when using CSMSHARELIB_CPL7 in install_lib_cpl7 mentioned below.